src/curated-corpus/components/RemoveItemFromScheduledSurfaceModal/RemoveItemFromScheduledSurfaceModal.test.tsx

```diff
@@ -1,32 +1,36 @@
 import React from 'react';
 import { render, screen, waitFor } from '@testing-library/react';
-import { ScheduledCorpusItem } from '../../../api/generatedTypes';
+import {
+  ScheduledCorpusItem,
+  ScheduledItemSource,
+} from '../../../api/generatedTypes';
 import { RemoveItemFromScheduledSurfaceModal } from './RemoveItemFromScheduledSurfaceModal';
 import { getTestApprovedItem } from '../../helpers/approvedItem';
-import { ScheduledSurfaces } from '../../helpers/definitions';
 import userEvent from '@testing-library/user-event';
 
 describe('The RemoveItemFromScheduledSurfaceModal component', () => {
   const approvedItem = getTestApprovedItem();
-  const germanItem: ScheduledCorpusItem = {
+  const pocketHitsItem: ScheduledCorpusItem = {
     approvedItem,
     createdAt: 1635014926,
     createdBy: 'jdoe',
     externalId: '123-test-id',
     scheduledDate: '2022-05-24',
-    scheduledSurfaceGuid: ScheduledSurfaces[1].guid,
+    scheduledSurfaceGuid: 'POCKET_HITS_DE_DE',
     updatedAt: 1635114926,
     updatedBy: 'jdoe',
+    source: ScheduledItemSource.Manual,
```

Comment:
This was missing and erroring in the IDE but tests run fine without the recently introduced required property.